src/css/theme/_geo-search.scss

```diff
@@ -43,6 +46,7 @@
   border: solid 1px #D4D8E3;
   border-radius: $border-radius;
   padding: 8px 15px;
+  outline: none;
```

Comment:
I don't think we should remove the outline without replacement though, see: [outlinenone.com](http://www.outlinenone.com)

Reply:
Not false. What alternative can you propose @Haroenv ?

Reply:
Completely agree! It's just a bad habits 😄

Reply:
1. don't change outline
2. we should have a proper outline style which isn't ugly in Chrome; cc @ronanlevesque